app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -210,31 +211,42 @@ fun SyncedTabsSectionHeader(
  */
 @Composable
 fun SyncedTabsTabItem(tabTitleText: String, url: String, onClick: () -> Unit) {
-    Column(
+    Row(
         modifier = Modifier
             .clickable(
                 onClickLabel = tabTitleText,
                 onClick = onClick
             )
             .padding(horizontal = 16.dp)
             .defaultMinSize(minHeight = 56.dp),
-        verticalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically,
```

Comment:
I see a 6dp top and 8dp bottom padding on inspection. What do you make of it versus using `CenterVertically`? Usually I go with the exact spec to align with what UX has designed.

<img width="1671" alt="Screen Shot 2022-03-09 at 6 17 02 PM" src="https://user-images.githubusercontent.com/1190888/157555320-d22b00d1-0a14-403b-a2cc-c5e762661756.png">

Reply:
the vertical alignment here is more so the icon is centered. If I use the padding values though, I may be able to remove the explicit `.defaultMinSize(minHeight = 56.dp)`